cluster-agent/utils/retrieve_metrics.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+// GET Request to retrive API Server metrics from the route endpoint
+func GetAPIServerMetrics(routeEndpoint string) string {
+	// #nosec G107
+	resp, err := http.Get(routeEndpoint)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	sb := string(body)
+	log.Printf(sb)
```

Comment:
Can remove the log statements that output to console, and just return the value below